joomla/joomla-cms
# [5.0] colour contrast in media manager file list [a11y]

Pull Request for Issue #42541 

### Summary of Changes
Fix a11y contrast issue in media manager

### Testing Instructions
Select an element in media, list view. The selected element has should have color white



### Actual result BEFORE applying this Pull Request
![image](https://github.com/joomla/joomla-cms/assets/1296369/bd82b3e6-2e51-4717-9682-5e98c030b6f9)



### Expected result AFTER applying this Pull Request
![image](https://github.com/joomla/joomla-cms/assets/1296369/26076c20-3af8-43a7-84f7-e523d43e624b)



### Link to documentations
Please select:
